### PR TITLE
Add additional support for network connectivity defined via configura…

### PR DESCRIPTION
**Pull request by pedro-r-marques**
…tion.

Global-networks allow a network to be connected to all pod networks.
Namespace-services determine pod network to service network connectivity on
a per namespace basis.

Change the default labels so that they are defined in 'opencontrail.org'
namespace.

### CODE REVIEW

File: pkg/network/opencontrail/config.go [hunk 1]
```diff
@@ -110,6 +124,21 @@ func validateClusterServiceName(name string) error {
 	return nil
 }
 
+func validateColonSeparatedNetworkName(name string) error {
+	networkName := strings.Split(name, ":")
+	if len(networkName) != 3 {
```
Comment (nati) [inline pkg/network/opencontrail/config.go]: I think this check fails with ":::" or "default-domain::network"

File: pkg/network/opencontrail/config.go [hunk 2]
```diff
@@ -110,6 +124,21 @@ func validateClusterServiceName(name string) error {
...
+		return fmt.Errorf("Expected 'domain:project:network', got \"%s\"", name)
+	}
+	return nil
+}
+
+func validateNamespaceService(name string) error {
+	if ix := strings.Index(name, "/"); ix != -1 {
```
Comment (nati) [inline pkg/network/opencontrail/config.go]: I perefer index than ix

Comment (nati) [inline pkg/network/opencontrail/config.go]: or just simply https://golang.org/pkg/strings/#Contains ?

File: pkg/network/opencontrail/controller_test.go [hunk 3]
```diff
@@ -556,7 +559,7 @@ func TestServiceDeleteWithPod(t *testing.T) {
 	obj, err = client.FindByName("floating-ip", strings.Join(sipName, ":"))
 	assert.Error(t, err)
 
-	obj, err = client.FindByName("network-policy", "default-domain:testns:x1")
+	obj, err = client.FindByName("network-policy", strings.Join(policyName, ":"))
 	assert.Error(t, err)
```
Comment (nati) [inline pkg/network/opencontrail/controller_test.go]: how about have a function for this such as toPolicyString() for readability?

Comment (nati) [inline pkg/network/opencontrail/controller_test.go]: or let Client also take string list format

File: pkg/network/opencontrail/policy.go [hunk 4]
```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2015 Juniper Networks, Inc. All rights reserved.
```
Comment (nati) [inline pkg/network/opencontrail/policy.go]: nit: 2016?

File: pkg/network/opencontrail/config.go (hunk 1 above)
Comment (pedro-r-marques) [inline pkg/network/opencontrail/config.go]: Done.

**Comment by nati** [comment]
Code looks good for me.
waiting CI get passed

**Pull request merged**